config/nginx/production/templates/default.conf

```diff
@@ -0,0 +1,37 @@
+# first we declare our upstream server, which is our Gunicorn application
+upstream django_server {
+    # docker will automatically resolve this to the correct address
+    # because we use the same name as the service: "django"
+    server  django:5000;
+}
+
+# now we declare our main server
+server {
+    listen  80;
+    server_name  localhost;
+    return 301 https://$host$request_uri;
+}
+
+server {
+    listen 443 ssl;
+    server_name localhost;
+
+    ssl_certificate           /etc/nginx/certs/cert.crt;
+    ssl_certificate_key       /etc/nginx/certs/cert.key;
+
+    ssl_session_cache  builtin:1000  shared:SSL:10m;
+    ssl_protocols  TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
Should only allow TLSv1.2 and TLSv1.3. all other versions are considered insecure.